### DISCUSSION TITLE
"Catalog must be specified when session catalog is not set"

### DISCUSSION THREAD

**Discussion by EvgSkv** [trinodb/trino#7748] [Q&A] [answered]
Sorry for a very basic question, but search doesn't give any clear answers.

I want to try Trino running locally. I created the basic configs from the set up tutorial.

But when I try to create schema I get a message about catalog, but I don't know how to create catalog.

```
trino> create schema my_schema;
Query 20210426_070146_00061_syvmn failed: line 1:1: Catalog must be specified when session catalog is not set
create schema my_schema
```

How should I proceed?

Thanks you!

**Comment by kokosing** [comment, marked as answer]
To create a catalog you need to place a properties file in `etc/catalog` directory and restart the Trino service.

Please also see https://trino.io/docs/current/overview/concepts.html#catalog to understand what catalog means in Trino.

Here is the most simple way to provide catalog with TPCH connector https://trino.io/docs/current/connector/tpch.html#configuration

After the above you can use one of the below to set the catalog:
 - `--catalog` in the Trino CLI (https://trino.io/docs/current/installation/cli.html)
 - `jdbc:trino://host:port/catalog` (https://trino.io/docs/current/installation/jdbc.html)
 - or by running `USE <catalog>.<schema>`; (https://trino.io/docs/current/sql/use.htm)

**Comment by EvgSkv** [comment]
Thank you very much for this fast and complete answer! It all works now for me.
